ui/src/app/layout/topics/single-topic.component.html

```diff
@@ -27,6 +27,14 @@ <h2 class="text-muted d-inline-block">{{ 'Topic' | translate }} {{ topicName | a
                                        <span class="deprecatedNote">Deprecated</span>
                                         <span class="deprecationText">{{(topic | async)?.deprecationText}}</span>
                                         </span>
+                                        <button
+                                            *ngIf="(isOwnerOfTopic | async) && !(selectedEnvironment | async)?.stagingOnly"
+                                            appSpinnerWhile
+                                            #p="appSpinnerWhile"
+                                            (click)="(openDeleteConfirmDlg(updateTopicDescription))"
```

Comment:
If you re-use `openDeleteConfirmDlg`for opening the dialog, you should rename that method ;-) Oh, and the outer braces can be removed.

Reply:
done.